examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -203,7 +203,7 @@ object SparkSQLExample {
     // No pre-defined encoders for Dataset[Map[K,V]], define explicitly
     implicit val mapEncoder = org.apache.spark.sql.Encoders.kryo[Map[String, Any]]
     // Primitive types and case classes can be also defined as
-    implicit val stringIntMapEncoder: Encoder[Map[String, Int]] = ExpressionEncoder()
+    // implicit val stringIntMapEncoder: Encoder[Map[String, Any]] = ExpressionEncoder()
```

Comment:
I am saying that the comments (L205) seem to indicate that, besides L204, the encoder can be defined as L206 as well. Actually, L206 has the incorrect type (you can find that by removing L204 and compile with the original L206)

so, I fixed the type and comment the line for successful compilation